components/user/UserMilestones.js

```diff
@@ -10,7 +10,8 @@ export default function UserMilestones({ milestones, manage = false }) {
   return (
     <>
       {!milestones?.length && (
-        <Alert type="info" message="No milestones found" />
+        <div className="m-6 mt-10"><Alert type="info" message="No milestones found" /></div>
```

Comment:
The other areas do not have it wrapped https://github.com/EddieHubCommunity/LinkFree/blob/main/components/user/UserTestimonials.js#L11

We should try to be consistent in the visual result and also the way it is implemented

Reply:
it was wrapped in the events tab. Check the highlights
![Screenshot 2023-07-05 114137](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/603ee6bb-6e2b-4b58-a5aa-2eba8ad66ee3)
this is for the changes i made on milestone
![milestone](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/a0235d2f-6c19-424c-a256-0e865f577a3e)

NOTE: The alert isnt popping up for the testimonials tab. the picture below shows what I'm getting with no testimonial but the alert message should be "No testimonials found". it doesn't look like the alert is popping up
![Screenshot 2023-07-05 115426](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/6bd65427-67eb-4219-afb8-41982bf4115f)

Reply:
Oh good spot, we should not show the `reorder` button if there are no results and should show the alert. As your PR is small, do you want to fix it as part of this?

Reply:
Yes I will love to. I'm guessing we are okay with the milestone?

Reply:
So I noticed, the UserTestimonial component (https://github.com/EddieHubCommunity/LinkFree/blob/main/components/user/UserTestimonials.js#L11) is not on the testimonials page (https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/account/manage/testimonials.js). Is it supposed to be that way?

Reply:
If it is that way, then the fix for this will be to show the Alert component only if there's no testimonial

Reply:
> If it is that way, then the fix for this will be to show the Alert component only if there's no testimonial

I have it working this way already.

Reply:
Hi, good morning. I'm still waiting on a response on this

Reply:
Reusing the component is just one line of code and I won't need to use the alert component. This is how it looks with the UserTestimonials component.
![testChange](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/1afc004e-7214-4c39-8a6c-4b9cecf0db0b)

this is the result (same result)
![change](https://github.com/EddieHubCommunity/LinkFree/assets/110388816/c089ab65-cd54-4293-ae45-dfb1bd645471)